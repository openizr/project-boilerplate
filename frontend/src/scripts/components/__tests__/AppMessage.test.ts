/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/vue';
import AppMessage from 'scripts/components/AppMessage.vue';

describe('components/AppMessage', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(AppMessage, { props: { label: 'test' } });
    expect(container.firstChild).toMatchSnapshot();
  });
});
