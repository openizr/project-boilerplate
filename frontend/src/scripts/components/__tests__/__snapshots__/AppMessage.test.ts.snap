// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AppMessage renders correctly - basic 1`] = `
<p>
  <img
    alt="test"
    src="file"
  />
  <img
    alt="test"
    src="file"
  />
   test
</p>
`;
